tic-tac-toe-app/GameBoard.jsx

```diff
@@ -0,0 +1,86 @@
+const calculateWinner = (board) =>{
+  const winningConditions = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ];
```

Comment:
Proactively setting up winning conditions helps to make the following code more clear. 👍